src/styles/index.js

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import * as PropTypes from 'prop-types';
```

Comment:
Why we are doing this? isn't PropTypes directly good enough?

Reply:
As PropTypes is no longer the default export name, we use * and change the name to PropTypes for better ide support

Reply:
Noted. Good one.